src/main/java/mat/server/service/FhirValidationReportService.java

```diff
@@ -129,7 +130,7 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
                         if("Not Found in Hapi".equals(results.getReason())) {
                             paramsMap.put("LibraryNotFoundInHapi", results.getReason());
                         }
-                        if (Boolean.FALSE.equals(results.getSuccess())) {
+                        if (results.getSuccess() == null || !results.getSuccess()) {
```

Comment:
According to Duane if results.getSuccues() is False, only then results.getLibraryFhirValidationResults() should be returning Errors. I am not sure if we should be considerring null.

Reply:
these are fixes so it can be null and still function. that was one of the issues we faced.